src/Sulu/Bundle/ContentBundle/Resources/config/search.xml

```diff
@@ -35,12 +35,9 @@
 
         <!-- Rebuild structure index on massive:search:index:rebuild -->
         <service id="sulu_content.search.event_listener.search" class="%sulu_content.search.event_listener.reindex.class%">
-            <argument type="service" id="sulu.phpcr.session" />
-            <argument type="service" id="sulu.content.mapper" />
+            <argument type="service" id="sulu_document_manager.document_manager" />
+            <argument type="service" id="sulu_document_manager.document_inspector" />
```

Comment:
I wonder if the document inspector should be available from the document manager